frontend/src/components/Pages/FrontPage/AutoScheduleSection/AutoScheduleCalendar.tsx

```diff
@@ -0,0 +1,239 @@
+                return {
+                    backColor: CalendarColors.ScheduledBackground,
+                    borderColor: CalendarColors.ScheduledBorder,
+                }
+            case MissionStatusType.FutureUnstartedJob:
+                return {
+                    backColor: CalendarColors.UpcomingBackground,
+                    borderColor: CalendarColors.UpcomingBorder,
+                }
+            case MissionStatusType.SkippedJob:
+                return {
+                    backColor: CalendarColors.SkippedBackground,
+                    borderColor: CalendarColors.SkippedBorder,
+                }
+            default:
+                return {
+                    backColor: CalendarColors.CompletedBackground,
+                    borderColor: CalendarColors.CompletedBorder,
+                }
+        }
+    }
+
+    const [calendarConfig, setConfig] = useState<CalendarProps>({
+        viewType: 'Days',
+        height: 600,
+        cellHeight: 35,
+        durationBarVisible: false,
+        headerDateFormat: 'dd dddd',
+        startDate: DayPilot.Date.today(),
+        days: 7,
+        timeFormat: 'Clock24Hours',
+        cellDuration: 30,
+        eventDeleteHandling: 'CallBack',
+        eventArrangement: 'SideBySide',
+        eventMoveHandling: 'Disabled',
+        onEventDelete: async (args) => {
+            const { text, start } = args.e.data
+            const missionId = args.e.data.data.tags.missionId
+            const timeOfDay = args.e.data.data.tags.timeOfDay
+
+            const dateString = DayPilot.Date.parse(start, 'yyyy-MM-ddTHH:mm:ss').toString('HH:mm')
+            const confirmed = window.confirm(
+                TranslateText('Are you sure you want to delete {0} scheduled for today at {1}?', [text, dateString])
+            )
+            if (!confirmed) return
+
+            await skipAutoScheduledMission(missionId, timeOfDay)
+            setConfig((prev) => ({
+                ...prev,
+                events: prev.events?.filter((e) => e.id !== args.e.id()) || [],
+            }))
+        },
+        onEventClicked: async (args) => {
+            const missionId = args.e.data.data.tags.missionId
+            navigate(`${config.FRONTEND_BASE_ROUTE}/mission-definition/${missionId}`)
+        },
+        events: [],
+    })
+
+    const events = useMemo(() => {
+        return timeMissionPairs.map(({ day, time, mission }) => {
+            const start = dateInDayPilotFormat(day, time)
+            const { backColor, borderColor } = choseEventColors(day, time, mission)
+
+            const isSkipped = selectMissionStatusType(day, time, mission) === MissionStatusType.SkippedJob
+            const displayText = isSkipped ? `${mission.name}` + ' (' + TranslateText('Skipped') + ')' : mission.name
+
+            return {
+                id: crypto.randomUUID(),
+                text: displayText,
+                start: start,
+                end: start.addMinutes(60),
```

Comment:
<img width="520" alt="image" src="https://github.com/user-attachments/assets/4f123a6d-0281-432c-8e5a-a8f69f5c702e" />

Reply:
13:00 and 13:01